gpib_server.py

```diff
@@ -176,10 +189,26 @@ def read(self, c, n_bytes=None):
         Otherwise, reads until the device stops sending.
         """
         instr = self.getDevice(c)
-        if n_bytes is None:
-            ans = instr.read_raw()
+        if instr.resource_name.startswith('USB'):
+            try:
+                if n_bytes is None:
+                    ans = instr.read()
+                else:
+                    ans = instr.read(n_bytes)
+            except:
+                self.asynchronous_wait(0.1)
+                if n_bytes is None:
+                    ans = instr.read()
+                else:
+                    ans = instr.read(n_bytes)
```

Comment:
I'd rather not have this delay/retry logic here in the gpib server, since it seems like a device-specific requirement. Ideally, this would go in the device server for the devices that need it. One thing we might want to do is add a setting on the gpib server that just does a delay, so that then the device server could send a single packet that includes the required delays, e.g.:
```python
p = cxn.gpib_bus.packet()
p.write(msg).delay(0.1).read()
result = yield p.send()
result.read
```
You could also do multiple writes with interleaved delays, etc, but all in one packet to the gpib bus server.

Similarly, we could add a delay parameter to the `query` setting, which would add a delay between the read and write in the same way.